radix-engine-monkey-tests/tests/fuzz_kernel.rs

```diff
@@ -471,7 +471,12 @@ fn kernel_fuzz<F: FnMut(&mut KernelFuzzer) -> Vec<KernelFuzzAction>>(
     let mut substate_db = InMemorySubstateDatabase::standard();
     let mut track = Track::new(&substate_db);
     let mut callback = TestCallbackObject;
-    let mut kernel = Kernel::new_no_refs(&mut track, &mut id_allocator, &mut callback);
+    let mut kernel = Kernel::new_no_refs(
+        &mut track,
+        &mut id_allocator,
+        &mut callback,
+        &ALWAYS_VISIBLE_GLOBAL_NODES_V2,
```

Comment:
I suggest we have a `const fn latest() -> Self` on `AlwaysVisibleGlobalNodesVersion` and use that to resolve the latest, rather than hard coding it to `V2`.

So this would be: `always_visible_global_nodes(AlwaysVisibleGlobalNodesVersion::latest())`.

Honestly, if  `Kernel::new_no_refs` is for tests, we could even just hard-code this into it and avoid changing all these signatures.